assets/os2forms_payment.js

```diff
@@ -37,7 +38,13 @@ function initPaymentWindow(checkoutContainer) {
     });
   };
   request.onerror = function () {
-    initPaymentWindow();
+    if (retried) {
+      alert("Der kunne ikke oprettes forbindelse til betalingsgatewayen. Prøv venligt igen senere.");
```

Comment:
This text should be translatable and be passed to the script from Drupal (hint: `data-payment-error-message="…"`).